.github/workflows/development-environment.yml

```diff
@@ -11,12 +11,17 @@ on:
 
 jobs:
   dev-environment:
-    name: python set up
+    name: set up
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 20
+    timeout-minutes: 25
     strategy:
       matrix:
-        os: [ macos-latest ]
+        # macosx-10.15 is Catalina
+        # macosx-11.0 is Big Sur, however, it takes long for jobs to get started
+        # Using Ubuntu 18 until I figure out this error:
+        # -> ImportError: libffi.so.6: cannot open shared object file: No such file or directory
+        os: [ macos-10.15, macos-11.0, ubuntu-18.04 ]
```

Comment:
Before I had `ubuntu-latest`, however, that can sometimes be 18.04 and sometimes 20.4. For now, let's pin it to one since 20.04 fails.